### ISSUE TITLE
 node-waf: command not found

### ISSUE THREAD

**Issue by goldenratio** [GrinZero/node-network-devtools#16]
Steps
- `npm add node-network-devtools`

I am getting below error

Error: 
```
npm error command failed
npm error command sh -c node-waf clean || true; node-waf configure build
npm error sh: node-waf: command not found
npm error sh: node-waf: command not found

```

My systems details:
node -v : v20.14.0
npm -v: 10.7.0
Mac OS: 14.5

**Comment by GrinZero** [comment]
Hello 👋 The problem has been identified as a dependency on the zlib library. I mistakenly used a persistent and unmaintained library, and I expect to fix this issue tonight.

**Comment by GrinZero** [comment]
Hello~The issue has been fixed in version v1.0.7, which has been released and is now available
